src/components/dropdownSearch/DropdownSearch.tsx

```diff
@@ -1,25 +1,25 @@
 import * as React from 'react';
 import { ISvgProps, Svg } from '../svg/Svg';
 import * as _ from 'underscore';
-import * as s from 'underscore.string';
 import * as classNames from 'classnames';
 import { FilterBox } from '../filterBox/FilterBox';
 import { keyCode } from '../../utils/InputUtils';
-import { FixedQueue } from '../../utils/FixedQueue';
+import * as s from 'underscore.string';
 
 export interface IDropdownOption {
   svg?: ISvgProps;
   value: string;
   displayValue?: string;
   prefix?: string;
+  selected?: boolean;
+  custom?: boolean;
+  hidden?: boolean;
```

Comment:
No because in the old dropdown search that PO made, an option can be both selected and displayed. But in dropdown search multiselect, a selected option is hidden in the dropdown.